runes_reforged_parser.py

```diff
@@ -0,0 +1,85 @@
+from collections import Counter
+import json
+import re
+
+import requests
+
+
+def guarantee_list(s):
+    if isinstance(s, list):
+        return s
+    if isinstance(s, str):
+        return re.findall(r"[^,]+", s)
+
+class RunesReforgedParser:
+    url = 'http://ddragon.leagueoflegends.com/cdn/10.16.1/data/en_US/runesReforged.json'
```

Comment:
nitpick: call this RUNES_REFORGED_DATA_URL or something (really doesn't matter though)